cou/steps/hypervisor.py

```diff
@@ -159,6 +160,16 @@ def get_azs(self, target: OpenStackRelease) -> AZs:
                 az = unit.machine.az or ""
                 azs[az].app_units[app.name].append(unit)
 
+            for s_unit in app.subordinate_units:
```

Comment:
I guess you are not considering that `ovn-chassis` can also relate with non-hypervisor applications. E.g: octavia.

How are you thinking to deal when that happens?